cg/apps/gisaid/gisaid.py

```diff
@@ -0,0 +1,125 @@
+"""Interactions with the gisaid cli upload"""
+
+import logging
+from pathlib import Path
+from typing import List
+
+from cg.store import models
+from ...meta.workflow.fastq import FastqHandler
```

Comment:
change to absolute imports
```suggestion
from cg.meta.workflow.fastq import FastqHandler
```